### DISCUSSION TITLE
so-allow automatic script

### DISCUSSION THREAD

**Discussion by b4b857f6ee** [Security-Onion-Solutions/securityonion#5514] [General] [answered]
Hello,

I can't find in the doc any information.
Can we script the add of the so-allow so we don't have to interact with the console.
My goal is to run a script using 2 input, (a,b,c,etc...) and the subnet to authorise a subnet/ip for a service in one shot instead of doing it with the console interactive menu.

Thank you for your help :)

**Comment by hacker0ni** [comment, marked as answer]
You can. Just use the flag with the so-allow command like so : `so-allow -a 192.168.1.1`

**Reply by b4b857f6ee** [reply]
OMG, you save me, thank you man!!! :D

**Reply by dougburks** [reply]
Updated documentation:
https://docs.securityonion.net/en/2.3/so-allow.html#automation
